ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -474,6 +514,40 @@ private Material GenerateGLTFMaterialFromRenderMaterial(VMaterial renderMaterial
             return material;
         }
 
+        private List<VAnimation> GetAllAnimations(VModel model)
+        {
+            var animGroupPaths = model.GetReferencedAnimationGroupNames();
+            var animations = model.GetEmbeddedAnimations().ToList();
+
+            // Load animations from referenced animation groups
+            foreach (var animGroupPath in animGroupPaths)
+            {
+                var animGroup = FileLoader.LoadFile(animGroupPath + "_c");
+                if (animGroup != default)
+                {
+                    var data = animGroup.DataBlock is ResourceTypes.NTRO ntro
```

Comment:
I believe we have a helper function that abstracts NTRO/BinaryKV3 getter.

Reply:
I can't find one? I took this implementation from `AnimationGroupLoader.cs`.

Reply:
Fixed as suggested via discord